generated/proto/pagetoken/pagetoken.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package pagetoken;
```

Comment:
would it be better to have this be `client` ?

Reply:
the package name? I'm open to renaming it, but client seems like a weird name because the token is actually opaque to the client, whereas its not opaque to the server